homework01/artur/vector.cc

```diff
@@ -0,0 +1,55 @@
+#include "vector.h"
+
+Vector::Vector()
+{
+}
+
+Vector::Vector(const int x, const int y)
+{
+  mX = x;
+  mY = y;
```

Comment:
Although in this particular case it doesnt change anything, its usually good to use initializer list